plugins/rbac-backend/src/service/policies-rest-api.ts

```diff
@@ -72,6 +81,7 @@ export class PoliciesServer {
     private readonly conditionalStorage: ConditionalStorage,
     private readonly pluginIdProvider: PluginIdProvider,
     private readonly roleMetadata: RoleMetadataStorage,
+    private readonly aLog: AuditLogger,
   ) {}
 
   private async authorize(
```

Comment:
Since the RBAC plugin is essentially serving as the permission backend plugin, are we not audit logging when a permission policy is requested and what the resultant decision was?

Reply:
We have component for permission evaluation `RBACPermissionPolicy`. That's custom permission handler. Backstage permission framework provides router with `/authorize` endpoint. This endpoint under the hood call this custom handler.